Dockerfile

```diff
@@ -1,6 +1,6 @@
 FROM alpine:3.21
 
-RUN apk add --no-cache ca-certificates && apk upgrade --update-cache --available
+RUN apk upgrade --update-cache --available && apk add ca-certificates && rm -rf /var/cache/apk/*
```

Comment:
noticed that the `add --no-cache` didn't make sense as the next statement added a cache anyway 😅